music/src/core/data.ts

```diff
@@ -69,8 +71,8 @@ export interface DrumsOneHotConverterSpec {
  * @property args Map containing values for argments to the constructor of the
  * `DataConverter` class specified above.
  */
-export type ConverterSpec = MelodyConverterSpec | DrumsConverterSpec |
-    DrumRollConverterSpec | TrioConverterSpec | DrumsOneHotConverterSpec;
+export type ConverterSpec = MelodyConverterSpec|DrumsConverterSpec|
```

Comment:
Did something happen with our clang-format configuration?  It used to put spaces here...

Reply:
Discussed offline.